pact_broker/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GEM
```

Comment:
Required due to trivy audit issues, performed by sourcing `scripts/docker-functions` and running `docker_build_bundle_base` & `bundle_update_on_docker`

my `DOCKER_DEFAULT_PLATFORM` is set to `linux/arm64` which may be worthy of note (although it shows no platform specific gems locked in here)